server/controllers/conversation.js

```diff
@@ -65,78 +44,61 @@ exports.getUserConversations = asyncHandler(async (req, res, next) => {
       success: true,
       conversations,
     });
-  } catch (error) {}
+  } catch (error) {
+    res.status(400).json({
```

Comment:
Please update status code to Server Error

Ref: https://www.restapitutorial.com/httpstatuscodes.html